versions.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# we will support at most two entries in each of these lists, and both should be in descending order
+supportedDebianSuites=(
+	bookworm
+)
+supportedAlpineVersions=(
+	3.18
+)
```

Comment:
Since they are defined here, can you make sure that `versions.sh` works with multiple suites/versions (and that the other scripts work for that as well)? I tried adding `bullseye`, but it's failing somewhere in `versions.sh`.

Reply:
I just tried and didn't have any issue with it

Reply:
It was failing regardless of if I tried to use these. The failure was on line 54.  I think `grep` on Debian doesn't understand `\d` except in PCRE mode. Swapping the pattern on 53 to `[0-9]` works fine for `-E` ("extended regular expressions").